operations/restrictions.md

```diff
@@ -395,6 +395,12 @@ This reduces the overall number of restrictions and improves system performance.
    1) If the new interval overlaps the old interval, the old restriction will be spliced before and after the new interval. Restrictions matching the old restriction are then added at the appropriate interval along with the new restriction.
    2) If the new interval does _not_ overlap the old interval, the new restriction is added as usual.
 
+### Affected restrictions
+
+Merging algorithm will also replace _all_ exceptions with exceptions defined in the request. It will also remove exception if null is sent. This can potentially remove exceptions defined in Mews but not supported in other applications.
+
+To avoid that, only restrictions created through the connector API are affected. Similiarly, if user creates restriction in Mews, the user will not affect restrictions created through the API.
```

Comment:
```suggestion
Only restrictions created through the API are affected by this operation, _not_ restrictions created by the user within **Mews Operations**. Similarly, if a user creates a restriction in **Mews Operations**, this will _not_ affect restrictions created through the API.
```